examples/resources/snowflake_user_grant/import.sh

```diff
@@ -0,0 +1,2 @@
+# format is user name | | | privilege | true/false for with_grant_option
+terraform import snowflake_user_grant.example 'userName|||MONITOR|true'
```

Comment:
what is with the weird id format?  why have `|||` instead of `|`?

Reply:
Actually this format is used everywhere, i'mjust trying to follow codebase format 
https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/72c318052ad6c29866cfee01e9a50a1aaed8f6d0/docs/resources/integration_grant.md?plain=1#L49-L50

https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/72c318052ad6c29866cfee01e9a50a1aaed8f6d0/docs/resources/account_grant.md?plain=1#L42-L43